tests/std/tests/P0645R10_text_formatting_formatting/test.cpp

```diff
@@ -841,6 +841,43 @@ void test_spec_replacement_field() {
     test_string_specs<charT>();
 }
 
+template <class charT, class... Args>
+void test_size_helper(size_t expected_size, basic_string_view<charT> fmt, const Args&... args) {
+    assert(formatted_size(fmt, args...) == expected_size);
+    assert(formatted_size(locale::classic(), fmt, args...) == expected_size);
+    basic_string<charT> str;
+    {
+        str.resize(expected_size);
+        auto res = format_to_n(str.begin(), expected_size, fmt, args...);
+        assert(res.size == expected_size);
+        assert(res.out - str.begin() == expected_size);
+        assert(res.out == str.end());

```

Comment:
Should we validate that the output of `format_to_n` is the same as the output of `format` since `n` is at least `expected_size` here? (Also on 860.)